cli/src/api/envs/list.rs

```diff
@@ -0,0 +1,12 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+/// Request body for /envs/list
+#[derive(Deserialize, Serialize)]
+pub struct EnvsListRequest {
```

Comment:
@quesabe I would get rid of `EnvsList` prefix. There is already a namespace defined in module path. Same for all other requests and responses.

What do you think?